jdaviz/configs/default/plugins/export/export.py

```diff
@@ -259,6 +277,11 @@ def export(self, filename=None, show_dialog=None):
                 raise NotImplementedError(f'Subset can not be exported - {self.subset_invalid_msg}')
             self.save_subset_as_region(selected_subset_label, filename)
 
+        elif len(self.dataset.selected):
+            filename = filename + ".fits"

```

Comment:
should have similar logic as tables/subsets to check if the extension is already provided and only add it otherwise

Reply:
Thanks, Kyle, for the feedback. Somehow, I thought we should only write data in a FITS file for now. Should I add a drop-down section for these formats [ASCII and FITS]?

Reply:
only fits is ok, but then we should still check if filename already ends with fits